Source/WebCore/dom/Document.cpp

```diff
@@ -3036,19 +3036,41 @@ bool Document::updateLayoutIfDimensionsOutOfDate(Element& element, OptionSet<Dim
 
     updateStyleIfNeeded();
 
+    if (layoutOptions.contains(LayoutOptions::ContentVisibilityForceLayout)) {
+        if (CheckedPtr renderer = element.renderer(); renderer &&  renderer->style().hasSkippedContent()) {
+            if (auto wasSkippedDuringLastLayout = renderer->wasSkippedDuringLastLayoutDueToContentVisibility()) {
+                if (*wasSkippedDuringLastLayout)
+                    renderer->setNeedsLayout();
+            }
+        }
+    }
+
```

Comment:
Do you mind landing this in a dedicated commit? -in case of the unthinkable  'IgnoreOverflow ' regression, it would be bad to lose this fix too.

Reply:
“Unthinkable”, indeed.

None of the existing callers pass the flag to get this behaviour (maybe they should?), but happy to do it separately regardless.